### PR TITLE
fix(server): wrong block reward

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/db/migrate/20190528034230_update_block_reward.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class UpdateBlockReward < ActiveRecord::Migration[5.2]
+  def up
+    current_epoch_number = api.get_current_epoch.number.to_i
```
Comment (classicalliu) [inline server/db/migrate/20190528034230_update_block_reward.rb]: does `current_epoch_number` can find from database?

Comment (shaojunda) [reply server/db/migrate/20190528034230_update_block_reward.rb]: yes, Thank you.

File: server/db/migrate/20190528034230_update_block_reward.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 class UpdateBlockReward < ActiveRecord::Migration[5.2]
   def up
+    return if Rails.env.test?
```
Comment (classicalliu) [inline server/db/migrate/20190528034230_update_block_reward.rb]: Any problem with test env ?

Comment (shaojunda) [reply server/db/migrate/20190528034230_update_block_reward.rb]: `CkbUtils.miner_reward(epoch)`  will call rpc, but there is no available node on test env

Comment (classicalliu) [reply server/db/migrate/20190528034230_update_block_reward.rb]: It's better to push this to task pool and not block migration.

Comment (shaojunda) [reply server/db/migrate/20190528034230_update_block_reward.rb]: ok, it make sense, I will move to task.

**Pull request merged**